iphone/Maps/Classes/CarPlay/Template Builders/ListTemplateBuilder.swift

```diff
@@ -89,13 +89,22 @@ final class ListTemplateBuilder {
   private class func obtainBookmarks(template: CPListTemplate, categoryId: MWMMarkGroupID) {
     let bookmarkManager = BookmarksManager.shared()
     let bookmarks = bookmarkManager.bookmarks(forCategory: categoryId)
-    let items = bookmarks.map({ (bookmark) -> CPListItem in
+    var items = bookmarks.map({ (bookmark) -> CPListItem in
       let item = CPListItem(text: bookmark.prefferedName, detailText: bookmark.address)
       item.userInfo = ListItemInfo(type: CPConstants.ListItemType.bookmarks,
                                    metadata: BookmarkInfo(categoryId: categoryId,
                                                           bookmarkId: bookmark.bookmarkId))
       return item
     })
+    if #available(iOS 15.0, *) {
+      let maxItemCount = CPListTemplate.maximumItemCount - 1
```

Comment:
Isn't it available from iOS 14? Why iOS 15 above?

Reply:
Disabling a cell is only available since iOS 15. If the cell isn’t disabled, the app crashes because it is looking for a bookmark that doesn’t exist.

Reply:
Will it work properly on iOS 12-14?

Reply:
Ok, that's an acceptable solution. So the fix applies only for iOS 15+, right? Let's update release notes )

Reply:
Yes, exactly only iOS 15+ sees the warning.